src/main/java/Duke/exception/DukeException.java

```diff
@@ -0,0 +1,7 @@
+package Duke.exception;
+
+public class DukeException extends Exception {
```

Comment:
This is a custom exception. Please include some Javadoc documentation to explain why this is there/what is its use in the codebase.